solution/ui/regulations/eregs-component-lib/src/api.js

```diff
@@ -255,7 +255,13 @@ const getSupplementalContent = async (
     );
 };
 
-const getFederalRegisterDocs = async (apiURL, { page = 1, pageSize = 3 }) => {
+const getRecentResources = async (apiURL, { page = 1, pageSize = 3 }, type) => {
+    if(type!=="rules"){
+        return httpApiGetLegacy(
+            `${apiURL}resources/supplemental_content?page=${page}&page_size=${pageSize}&paginate=true`,
+            {}, // params, default
+            apiURL);
+    } 
```

Comment:
since we're refactoring this API call, it would be nice to move it out to the shared `api.js` file in the parent directory, since we'll need to do that eventually anyway.

Reply:
moved